### PR TITLE
Added my details to data.json.

### LINKED ISSUE #598: Siddhi Bhanushali
Screenshot_20210718-090707_Chrome

### PR DESCRIPTION
**Pull request by siddhi-244**
Fixes #598

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1126,5 +1126,12 @@
     "githubUsername": "hill",
     "imageUrl": "https://user-images.githubusercontent.com/5620045/125885321-8cda4575-b72f-42ea-99ad-4de890e747ad.png",
     "issueNumber": 592
-  }
+  },
+  {
+
```
Comment (schmelto) [inline data.json:1131]:
```suggestion
```

File: data.json [hunk 2]
```diff
@@ -1126,5 +1126,12 @@
...
     "issueNumber": 592
-  }
+  },
+  {
+    
+    "name" : "Siddhi Bhanushali ",
+    "githubUsername" : "siddhi-244",
+    "imageUrl" : "https://user-images.githubusercontent.com/69195262/126054873-11415026-bcee-40d4-81dc-db3076965670.jpg",
+    "issueNumber" : 598
+    }
```
Comment (schmelto) [inline data.json:1136]: Please align with the other brackets

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by siddhi-244** [comment]
@schmelto  done! Please check !

**Comment by kaiwalyakoparkar** [comment]
The prettier is failing for the files which were not changed in this commit. :thinking: Is this a normal issue? @siddhi-244 Did you try running prettier in your local machine while making the changes?

**Comment by siddhi-244** [comment]
> The prettier is failing for the files which were not changed in this commit. 🤔 Is this a normal issue? @siddhi-244 Did you try running prettier in your local machine while making the changes?

I saw that it failed during other people's time as well.

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket: Thanks for collaborating and contributing

**Comment by siddhi-244** [comment]
> LGTM 🚀 Thanks for collaborating and contributing

Thank you !

**Comment by HareerSakhi** [comment]
@kaiwalyakoparkar Maybe we should merge this PR soon before there are too many conflicts?

**Comment by kaiwalyakoparkar** [comment]
Oh! @HareerSakhi you are right :frowning: It created conflicts. As soon as @siddhi-244 fixes the conflicts. I or @schmelto will merge it.

**Comment by siddhi-244** [comment]
> Oh! @HareerSakhi you are right 😦 It created conflicts. As soon as @siddhi-244 fixes the conflicts. I or @schmelto will merge it. Thanks for looking through the PR's

Sure I will !

**Pull request merged**